utils/gyb.py

```diff
@@ -1042,15 +1042,19 @@ def execute_template(
     ... THIS SHOULD NOT APPEAR IN THE OUTPUT
     ... ''')
     >>> out = execute_template(ast, line_directive='//#sourceLocation', x=1)
-    >>> out = out.replace(os.path.abspath(os.sep) + 'dummy.file', "DUMMY-FILE")
+    >>> # '/dummy.file' gets resolved to an absolute path, and is prended by
+    >>> # '/' on Unix/macos or 'C:/' on Windows. We need to adjust for this
+    >>> # difference.
+    >>> root_directory = os.path.abspath(os.sep)
+    >>> out = out.replace(root_directory, "ROOT")
```

Comment:
This didn't work on Unix - as we ended up replacing each `/` in the `// comment` bit.
I've tried another approach. This is a hard nut to crack, sorry